tools/lint/rules.py

```diff
@@ -350,6 +350,13 @@ class LayoutTestsRegexp(Regexp):
     file_extensions = [".html", ".htm", ".js", ".xht", ".xhtml", ".svg"]
     description = "eventSender/testRunner/internals used; these are LayoutTests-specific APIs (WebKit/Blink)"
 
+class MissingDepsRegexp(Regexp):
+    pattern = br"[^\w]/gen/"
```

Comment:
Hmm so in theory we can probably do better than a regexp here, but it's also likely not worth it.